### PR TITLE
LPS-172142: POST JSON call response's body is not correctly parsed

### PR DESCRIPTION
**Pull request by sergiojimcos**
**What is new?**
- Changes in `dto.ftl`, checking if `parameterType` is `Long` or `Integer`, it will use the `@Min` and `@Max` annotations. For BigDecimals and Double `parameterType` it will maintain the `@DecimalMin` and `@DecimalMax` annotation.
- `buildREST` in `modules/apps` and `modules/dxp/apps`.

**Before PR**
image

**After PR**
image

**JIRA Ticket**
https://issues.liferay.com/browse/LPS-172142

### CODE REVIEW

**Comment by sergiojimcos** [comment]
ci:test:relevant

File: modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/dto.ftl [hunk 1]
```diff
@@ -142,7 +143,11 @@ public class ${schemaName} <#if dtoParentClassName?has_content>extends ${dtoPare
 		</#if>
```
Comment (ccorreagg) [inline modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/dto.ftl:143]: What about modifing the max?

Comment (sergiojimcos) [reply modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/dto.ftl:143]: I could do it, I don't know if there is any expected error for that but I don't mind to change it

Comment (ccorreagg) [reply modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/dto.ftl:143]: If that is the error with the minimum functionality, then we must apply the change also to the maximum

Comment (ccorreagg) [reply modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/dto.ftl:143]: Just curious, do we have the error for Double parameters or it is not reproducible for that type?

Comment (sergiojimcos) [reply modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/dto.ftl:143]: I will check that :grin:

**Comment by sergiojimcos** [comment]
ci:test:sf

**Comment by sergiojimcos** [comment]
ci:test:relevant

File: modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/dto.ftl (hunk 1 above)
Comment (ccorreagg) [reply modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/dto.ftl:148]: but if there is code for supporting the Double, we should check if it works; if there is no double in Liferay's REST Builder APIs, maybe we could test it by just including a new field in an existing API and validating it

**Comment by sergiojimcos** [comment]
Closign this PR, see this https://github.com/liferay-headless/liferay-portal/pull/1006

**Pull request closed without merging**